md/database-configuration.md

```diff
@@ -237,6 +237,22 @@ See [in-doubt xact resolution Server Configuration Option](https://msdn.microsof
 
 #### MySQL
 
+##### XA Transactions
+
+To support XA transactions, starting from MySQL 8.0, special XA rights must be granted to the database user. For example, if the users for the engine and BDM databases are `bonita` and `business_data` respectively, you should run the following command:
+
+```sql
+GRANT XA_RECOVER_ADMIN ON *.* to bonita, business_data;
```

Comment:
I don't understand... I am convinced I *did* this contribution already, when I discovered this new grant requirement for 7.9 .... :thinking: ...